modules/logging-loki-cli-install.adoc

```diff
@@ -76,7 +76,13 @@ spec:
   tenants:
     mode: openshift-logging
 ----
-<1> Supported size options for production instances of Loki are `1x.small` and `1x.medium`.
++
+[NOTE]
+====
+In the {logging-title} 5.8 and later versions, the `1x.extra-small` Loki instance size is a supported option for production environments.
+====
++
```

Comment:
Is the note really necessary here since the supported versions are listed in the callout below?